src/EventStore.Core/TransactionLog/Chunks/TFChunkScavenger.cs

```diff
@@ -112,8 +117,8 @@ public long Scavenge(bool alwaysKeepScavenged, bool mergeChunks)
                             mergedSomething = true;
                         }
                     }
-                    Log.Trace("SCAVENGING: merge pass #{0} completed in {1}. {2} merged.",
-                              passNum, sw.Elapsed, mergedSomething ? "Some chunks" : "Nothing");
+                    Log.Trace("SCAVENGING: {0}", string.Format("merge pass #{0} completed in {1}. {2} merged.",
```

Comment:
There should be no need for the `string.Format` here - IIRC the loggers format strings in the same manner.